src/playwright/testplan.ts

```
@@ -34,16 +36,67 @@ export class TestPlan {
   getTestItems(): TestItem[] {
-    return this.templates.map((template: string) => {
-      return new TestItem(
-        template as unknown as TemplateType,
-        this.tssc.registry,
-        this.tssc.git,
-        this.tssc.ci,
-        this.tssc.tpa,
-        this.tssc.acs
-      );
+    const testItems: TestItem[] = [];
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)